filament/include/filament/Engine.h

```diff
@@ -54,6 +54,22 @@ class LightManager;
 class RenderableManager;
 class TransformManager;
 
+#ifndef FILAMENT_PER_RENDER_PASS_ARENA_SIZE_IN_MB
+// Froxelization needs about 1 MiB. Command buffer needs about 1 MiB.
+#    define FILAMENT_PER_RENDER_PASS_ARENA_SIZE_IN_MB 3
+#endif
+
+#ifndef FILAMENT_PER_FRAME_COMMANDS_SIZE_IN_MB
+// size of the high-level draw commands buffer (comes from the per-render pass allocator)
+#    define FILAMENT_PER_FRAME_COMMANDS_SIZE_IN_MB 2
+#endif
+
+#ifndef FILAMENT_MIN_COMMAND_BUFFERS_SIZE_IN_MB
+// size of a command-stream buffer (comes from mmap -- not the per-engine arena)
+#    define FILAMENT_MIN_COMMAND_BUFFERS_SIZE_IN_MB 1
+#endif
+
+
```

Comment:
I don't think we need this in the public header.

Reply:
Ok.  I'm using them as default values for parameters.  I will replace them with straight literals.  I'll continue to use the #define's inside the functions.

Reply:
Actually you're right, after you make the change where `Config` is a simple struct with default values, you'll need those in the public header -- so let's keep them here.